lang/rust/Makefile

```diff
@@ -0,0 +1,182 @@
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rust
+PKG_VERSION:=1.66.1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=Apache-2.0 MIT
+PKG_LICENSE_FILES:=LICENSE-APACHE LICENSE-MIT
+
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/rustc-$(PKG_VERSION)-src/
+PKG_SOURCE:=rustc-$(PKG_VERSION)-src.tar.gz
+PKG_SOURCE_URL:=https://static.rust-lang.org/dist/
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_HASH=5b3c933a94c72187705d4ee293198babfdd09442f5937fbd685db3a81f4959ba
+
+PKG_HOST_ONLY:=1
+PKG_BUILD_PARALLEL:=1
+
+HOST_USE_NINJA:=1
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/cmake.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+
+# Requires Python3 to build
+HOST_BUILD_DEPENDS:=python3/host
```

Comment:
There is also a need to `ninja/host`

Reply:
It does not exist.

Reply:
Ho sorry, I did it on an old openwrt version, ninja package was removed from my old version